packages/itwinui-react/src/core/utils/functions/dev.ts

```diff
@@ -2,13 +2,13 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-const isJest = typeof jest !== 'undefined';
+const isVitest = typeof vitest !== 'undefined';
```

Comment:
Undo this. This is not related to our tests, it's part of our runtime code where we actually want to check for `jest`.

Reply:
> Done ([399a837](https://github.com/iTwin/iTwinUI/commit/399a83713910318bbe0086b3ab0e91faf6690861)).

It seems incorrect that `isJest` is checking whether we're in `vitest`. If we want to specifically check for `jest` in our runtime code, we can maybe have both `isJest` and `isVitest`.

> Since I upgraded the [ testing-library/jest-dom](https://github.com/testing-library/jest-dom#with-vitest) to latest version
> and added this import statement `import '@testing-library/jest-dom/vitest'; I am getting the error in the above file as shown in ss.

When I ran `yarn test` in `@itwin/itwinui-react` on the latest commit in this branch (https://github.com/iTwin/iTwinUI/pull/1784/commits/3a51948e0adf4b35cadba7e6f4eeff597a167f25), I didn't get this error.

Reply:
We should have a common reliable check that detects jest, vitest, and mocha

For now, you should be able to work around the error by chaging `typeof `jest` to `typeof globalThis.jest`

Reply:
Yeah, ts-ignore is fine here

Reply:
Lets handle `isJest`/`isDev` issues in a separate follow-up PR, since it affects runtime code

Reply:
Sounds good. @siddhantrawal, maybe you can add a bullet point TODO in the PR description so that we can remember to address this after the PR merge.